packages/flutter/lib/src/material/floating_action_button_theme.dart

```diff
@@ -96,6 +101,25 @@ class FloatingActionButtonThemeData with Diagnosticable {
   /// ignored.
   final bool? enableFeedback;
 
+  /// The size constraints for the [FloatingActionButton].
```

Comment:
It's slightly more correct to say "overrides the default size constraints for the ..." (here and below).

Reply:
Done